cron-jobs/sourceballs

```diff
@@ -56,19 +56,13 @@ done | sort -u > "${WORKDIR}/available-src-pkgs"
 for repo in ${PKGREPOS[@]}; do
 	newpkgs=()
 	failedpkgs=()
-	while read line; do
-		pkginfo=(${line})
-		pkgbase=${pkginfo[0]}
-		pkgver=${pkginfo[1]}
-		pkgarch=${pkginfo[2]}
-		pkglicense=(${pkginfo[@]:3})
-
+	while read -r pkgbase pkgver pkgarch pkglicense; do
```

Comment:
pkglicense is now a string, rather than an array. It would still work as both licenses are in the (unquoted) array element 0, but of course that is now quoted.

So this only works if there is exactly one license in play.